src/python/pcmdi/mean_climate_metrics_calculations.py

```diff
@@ -43,7 +43,7 @@ def compute_metrics(Var, dm, do):
 
     # SET CONDITIONAL ON INPUT VARIABLE
     if var == 'pr':
-        conv = 1.e5
+        conv = 86400. 
```

Comment:
@durack1 @gleckler1 this is due to some `bad units`  we should probably enforce S.I units as a requirement and drop this.